bsd/disk.pyx

```diff
@@ -5,6 +5,28 @@ from fcntl import ioctl
 from . cimport disk_ioctl
 
 
+def get_sectorsize_with_name(disk):
+    """
+    Get sectorsize of disk.
+
+    `disk` must be a string and can be `/dev/da1` or just `da1`
+    """
+    disk = disk.removeprefix('/dev/')
+    with open(f'/dev/{disk}', 'rb') as f:
```

Comment:
Is this always guaranteed to exist ?

Reply:
This is _EXACTLY_ formatted the same way as the other methods. I don't feel like making this method different than the others.